### DISCUSSION TITLE
Resource View: Having horizontal scroll

### DISCUSSION THREAD

**Discussion by rbmnex** [vkurko/calendar#242] [Q&A]
In my project, there will be more that 10+ staff need display on resource view but display will cramp in mobile, but if increase width is hard to navigate in mobile view. Any suggestion?

**Comment by vkurko** [comment]
The solution would be [Resource view - horizontal](https://github.com/vkurko/calendar/issues/30), but unfortunately this has not yet been implemented.

**Comment by stescacom** [comment]
Okay So I was playing around and wanted to achieve the horizontal scroll... This is how I achieved it.

First in your CSS

```
.ec-days {
    max-width: 500px;
    overflow-x: auto;
}

.ec-day {
    min-width: 150px;
}

```

Next... we need to keep the scroll synchronous so in your `JS`

```
let els = document.getElementsByClassName('ec-days');
for (let el of els) {
    el.addEventListener('scroll', function (evt) {
        for (let lel of els) {
            if (lel != evt.target) {
                lel.scrollLeft = evt.target.scrollLeft;
            }
        }
    });
}

```

This is just a rough test.. you can optimize your JS. Hope it helps.
